src/mmw/apps/modeling/tasks.py

```diff
@@ -146,8 +146,19 @@ def analyze_pointsource(area_of_interest):
     return [point_source_pollution(area_of_interest)]
 
 
+def ensure_aoi_4d(area_of_interest):
```

Comment:
Given that you had trouble reproducing the MultiPolygon generation from the app, this is probably a good candidate to create a unit test for, to ensure it produces correct values for both `Polygon` and `MultiPolygon` input.  You should use the actual json argument captured by RollBar as the input for the Multi.

This django app has the testing infrastructure in place:
https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/tests.py